orc/graphs/main.py

```diff
@@ -49,7 +51,7 @@ def create_main_agent(
             "AZURE_OPENAI_API_VERSION", "2024-05-01-preview"
         ),
         azure_deployment="gpt-4o-orchestrator",
-        model_kwargs={"seed": 1},
+        seed = 1,
```

Comment:
good job with the setup of the set-up of this instance. just wondering why you changed this from `model_kwargs = {"seed": 1}` to `seed = 1`. is the latter a more efficient way for the initialization of seed?